deploy/06_deploy_coverage_pool.ts

```diff
@@ -14,40 +13,20 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
     log: true,
   })
 
-  if (
-    helpers.address.equal(
-      await read("AssetPool", "owner"),
-      CoveragePool.address
-    )
-  ) {
-    log(`transferring ownership of AssetPool to ${CoveragePool.address}`)
-
-    await execute(
-      "AssetPool",
-      { from: deployer },
-      "transferOwnership",
-      CoveragePool.address
-    )
-  }
-
-  if (
-    helpers.address.equal(
-      await read("UnderwriterToken", "owner"),
-      AssetPool.address
-    )
-  ) {
-    log(`transferring ownership of UnderwriterToken to ${AssetPool.address}`)
-
-    await execute(
-      "UnderwriterToken",
-      { from: deployer },
-      "transferOwnership",
-      AssetPool.address
-    )
-  }
+  await helpers.ownable.transferOwnership(
+    "AssetPool",
+    CoveragePool.address,
+    deployer
+  )
+
+  await helpers.ownable.transferOwnership(
+    "UnderwriterToken",
+    AssetPool.address,
+    deployer
+  )
```

Comment:
> In this context, I do not see where do we transfer RewardsPool ownership to CoveragePoolBeneficiary 🤔

198b5e164f343d3fa2e7d32e45ae1ee989b90ca9

Reply:
> Wouldn't it be better to have all ownership transfers aggregated in a separate step, just like we have for community multisig transfer in step 14? Less chance we'll miss something.

As we separated scripts for contracts to separate files to easier track them and make use of tags and dependencies I believe it's better to keep the ownership transfers in the individual places rather than gathering them all together.

Reply:
In that case, shouldn't we have
https://github.com/keep-network/coverage-pools/blob/063db74969fed5c6ff75455d4487ba2150f424f4/deploy/06_deploy_coverage_pool.ts#L22-L27

in `05_deploy_asset_pool.ts` instead of in `06_deploy_coverage_pool.ts`?

Reply:
d19606cf8fb666d27c1fd9be2483f9cf54007839